playbooks/common/openshift-cluster/config.yml

```diff
@@ -17,6 +17,24 @@
       - package_version
       - docker_image_availability
       - docker_storage
+  when: ansible_distribution != "Fedora"
+
+- name: Verify Requirements - Fedora
+  hosts: oo_all_hosts
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: install
+  post_tasks:
+  - action: openshift_health_check
+    args:
+      checks:
+      - disk_availability
+      - memory_availability
+      - package_availability
+      - docker_image_availability
+      - docker_storage
+  when: ansible_distribution == "Fedora"
```

Comment:
@rhcarvalho I understand there's nothing distro specific about the check, what I meant was: Is this a change we want for all distros or just Fedora? Because if it's not Fedora specific, then it's out of scope for this Pull Request.